### PR TITLE
feat: added scripts and backend routes for deployment V1

### PR DESCRIPTION
**Pull request by brian-walvoord**

### CODE REVIEW

**Review by calss0t — APPROVED** [review]
Can you explain Tuesday what this changes do? I'm not sure I understand.
But looks good, so approved

**Review by Takahiro9-Murakami9 — COMMENTED** [review]
As pol said, please explain what you changed. I want to understand. But it's good👍😊I want approve it, but approve selectors didn't appear. Why?

File: backend/server.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ require("dotenv").config({path: "./.env.local"});
 const API_KEY = process.env.API_KEY_TMDB
 
 
+// have node serve the files for our built React app
+const path = require('path');
```
Comment (Takahiro9-Murakami9) [inline backend/server.js:14]: Please explain this line.

**Comment by brian-walvoord** [comment]
@Takahiro9-Murakami9 I requested everybody because I wasn't sure who was available, but Pol already merged it is why.

**Comment by Takahiro9-Murakami9** [comment]
Ok! I understood!

**Pull request merged**